src/transformers/models/bridgetower/image_processing_bridgetower.py

```diff
@@ -161,29 +161,53 @@ class BridgeTowerImageProcessor(BaseImageProcessor):
         do_pad (`bool`, *optional*, defaults to `True`):
             Whether to pad the image to the `(max_height, max_width)` of the images in the batch. Can be overridden by
             the `do_pad` parameter in the `preprocess` method.
+        pad_and_return_pixel_mask (`bool`, *optional*):
+            Deprecated. Whether to pad the image to the `(max_height, max_width)` of the images in the batch.
+            Sets do_pad.
+        return_tensors (`str` or `TensorType`, *optional*):
+            The type of tensors to return. Can be one of:
+            - Unset: Return a list of `np.ndarray`.
+            - `TensorType.TENSORFLOW` or `'tf'`: Return a batch of type `tf.Tensor`.
+            - `TensorType.PYTORCH` or `'pt'`: Return a batch of type `torch.Tensor`.
+            - `TensorType.NUMPY` or `'np'`: Return a batch of type `np.ndarray`.
+            - `TensorType.JAX` or `'jax'`: Return a batch of type `jax.numpy.ndarray`.
+        data_format (`ChannelDimension` or `str`, *optional*, defaults to `"channels_first"`):
+            The channel dimension format for the output image. Can be one of:
+            - `"channels_first"` or `ChannelDimension.FIRST`: image in (num_channels, height, width) format.
+            - `"channels_last"` or `ChannelDimension.LAST`: image in (height, width, num_channels) format.
+            - Unset: Use the channel dimension format of the input image.
+        input_data_format (`ChannelDimension` or `str`, *optional*):
+            The channel dimension format for the input image. If unset, the channel dimension format is inferred
+            from the input image. Can be one of:
+            - `"channels_first"` or `ChannelDimension.FIRST`: image in (num_channels, height, width) format.
+            - `"channels_last"` or `ChannelDimension.LAST`: image in (height, width, num_channels) format.
+            - `"none"` or `ChannelDimension.NONE`: image in (height, width) format.
     """
 
     model_input_names = ["pixel_values"]
 
     def __init__(
         self,
         do_resize: bool = True,
-        size: Dict[str, int] = 288,
+        size: Dict[str, int] = {"shortest_edge": 288},
         size_divisor: int = 32,
         resample: PILImageResampling = PILImageResampling.BICUBIC,
         do_rescale: bool = True,
         rescale_factor: Union[int, float] = 1 / 255,
         do_normalize: bool = True,
         image_mean: Optional[Union[float, List[float]]] = None,
         image_std: Optional[Union[float, List[float]]] = None,
-        do_center_crop: bool = True,
-        do_pad: bool = True,
+        do_center_crop: Optional[bool] = True,
+        do_pad: Optional[bool] = True,
```

Comment:
These shouldn't be optional in the init 

```suggestion
        do_center_crop: bool = True,
        do_pad: bool = True,
```

Reply:
absolutely, fixed in https://github.com/huggingface/transformers/pull/28843 which should be merged before that one